.github/workflows/build.yml

```diff
@@ -374,6 +370,19 @@ jobs:
       - name: Build COBOL LS extension 
         run: npm ci
         working-directory: clients/cobol-lsp-vscode-extension
+      - name: Inject telemetry key to COBOL LS extension
+        shell: pwsh
+        run: |
+          $telKey = node -e "console.log(Buffer.from('${{ secrets.TELEMETRY_INSTRUMENTATION_KEY }}').toString('base64'))"
```

Comment:
a simple `sed -i` command?

Reply:
That was my first choice, but this step runs on all different platforms and the `sed` version mac is not compatible with linux `sed`. This powershell version seems to be running on all platforms reliably.